src/js/game/hud/parts/building_placer_logic.js

```diff
@@ -467,7 +467,14 @@ export class HUDBuildingPlacerLogic extends BaseHUDPart {
                 index = 0;
                 console.warn("Invalid variant selected:", this.currentVariant.get());
             }
-            const newIndex = (index + 1) % availableVariants.length;
+            const direction = this.root.keyMapper.getBinding(KEYMAPPINGS.placement.rotateInverseModifier)
+                .pressed
+                ? -1
+                : 1;
+            let newIndex = (index + direction) % availableVariants.length;

```

Comment:
see above.

Also don't use `rotateInverseModifier` -> Instead lets add a new bindign

Reply:
Another key? Omg...
Same feature for Cycle buildings("Tab")?
So boring to cross half of screen, just to pick trash can...
Or ability to use scroll wheel with an extra binding(ofcource different from one used here :) )

Reply:
I personally feel that adding another binding would be excessive - I've always treated `(E ->) TAB -> T -> R` as a group of bindings that let you navigate the building options, and I think a single `backwardsModifier` or something would make the most sense

Reply:
Yeah I think we should unify it into one binding

Reply:
Should rotateInverseModifier be changed to something else, and/or should the current translations of it be reverted?

Reply:
I'd:
1) Get rid of rotateInverseModifier
2) Introduce a new binding "Backwards modifier"